modules/thermal_hydraulics/include/base/WallFrictionModels.h

```diff
@@ -34,4 +34,17 @@ ADReal DarcyFrictionFactor(const ADReal & f_F);
  */
 Real FanningFrictionFactorChurchill(Real Re, Real roughness, Real D_h);
 ADReal FanningFrictionFactorChurchill(ADReal Re, ADReal roughness, ADReal D_h);
+
+/**
+ * Computes Fanning friction factor using Cheng-Todreas correlation
+ *
+ * @param Re   Reynolds number
+ * @param a    Correlation constant
+ * @param b    Correlation constant
+ * @param c    Correlation constant
+ * @param n    Correlation constant
+ * @param PoD  Pitch-to-diameter ratio
+ */
+Real FanningFrictionFactorCheng(Real Re, Real a, Real b, Real c, Real n, Real PoD);
+ADReal FanningFrictionFactorCheng(ADReal Re, ADReal a, ADReal b, ADReal c, ADReal n, ADReal PoD);
```

Comment:
template

Reply:
here we go again hahaha

Reply:
So, I also write the other friction factors as templates and the WallFrictionModels.C file ended up "empty", can I just get rid of it?

Reply:
Yep, you can delete.

Reply:
Ok, I tried once after deleting the WallFrictionModels.C and the tests here returned a bunch of errors, I'm trying again with only the Peclet as template, and will do the others latter this week or in the next one (I will keep working on THM stuff after the end of my internship), Lise and I want to finish the new Closure before my last day at INL

Reply:
Sounds good - if it's too much trouble, don't worry about it.